src/FixedDataTableCellGroup.js

```diff
@@ -164,44 +161,12 @@ class FixedDataTableCellGroup extends React.Component {
       this._staticCells[i] = this._renderCell(i, columnIndex);
     }
 
-    var style = {
-      height: props.cellGroupWrapperHeight || props.height,
-      position: 'absolute',
-      width: props.contentWidth,
-      zIndex: props.zIndex,
-    };
-
-    // // // // console.log(this._initialRender,"Hello")
-    FixedDataTableTranslateDOMPosition(
-      style,
-      -1 * props.left,
-      0,
-      this._initialRender,
-      this.isRTL
-    );
-
-    if (this.props.isRTL) {
-      style.right = props.offsetLeft;
-    } else {
-      style.left = props.offsetLeft;
-    }
-
     // NOTE (pradeep): Sort the cells by column index so that they appear with the right order in the DOM (see #221)
     const sortedCells = _.sortBy(this._staticCells, (cell) =>
       _.get(cell, 'props.columnIndex', Infinity)
     );
 
-    return (
-      <div
-        // className={joinClasses(cx('fixedDataTableCellGroupLayout/cellGroup'),cx('fixedDataTableCellGroupLayout/cellGroupWrapper'))}
-        style={style}
-      >
-        {/* <div> */}
-        {/* <> */}
-        {sortedCells}
-        {/* </> */}
-      </div>
-    );
+    return <>{sortedCells}</>;
```

Comment:
We can remove this div now .I have also included this class name to the cell for maintaining the CSS associated with it